ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/component/StateComponent.java

```diff
@@ -41,6 +41,6 @@
      *
      * @param state the component state object
      */
-    void loadState(@NotNull JsonObject state);
+    void setState(@NotNull JsonObject state);
```

Comment:
according to JavaDoc `Called when component should restore his state`. loadSatte more reasonable for me.
@evidolob your opinion?